core/src/utils/mod.rs

```diff
@@ -1 +1,18 @@
 pub mod test;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::io::Error;
+use std::path;
+
+pub fn store_json_data(data: String, dest_folder: String, dest_path: &String) -> Result<(), Error> {
+    let path = path::Path::new(&dest_folder);
+    if !path.exists() {
+        fs::create_dir_all(&dest_folder)?
```

Comment:
It disturbed me to not see `;` at the end of this line, although it seems to work fine.
Is there any reason to omit it ?
```suggestion
        fs::create_dir_all(&dest_folder)?;
```

Reply:
Haha I feel you. We could may add that check to clippy on a different PR